pkg/ocm/managedservice.go

```diff
@@ -22,6 +22,10 @@ type CreateManagedServiceArgs struct {
 	AwsRegion              string
 
 	AwsOperatorIamRoleList []OperatorIAMRole
+
+	MultiAZ              bool
+	AwsAvailabilityZones []string
```

Comment:
I would rename this to `AvailabilityZones`, since everything in ROSA is implied AWS already.